diesel_derives/tests/selectable.rs

```diff
@@ -0,0 +1,99 @@
+use diesel::*;
+
+use helpers::connection;
+
+#[cfg(feature = "mysql")]
+type IntSql = ::diesel::sql_types::BigInt;
+#[cfg(feature = "mysql")]
+type IntRust = i64;
+
+#[cfg(not(feature = "mysql"))]
+type IntSql = ::diesel::sql_types::Integer;
+#[cfg(not(feature = "mysql"))]
+type IntRust = i32;
```

Comment:
This is not required anymore. We can now just use `i64`/`BigInt` for all backends